src/main.cpp

```diff
@@ -13,7 +33,7 @@ int main() {
 
 	bot.on_log(dpp::utility::cout_logger());
 
-	bot.on_slashcommand([](const dpp::slashcommand_t& event) {
+	bot.on_slashcommand([&bot](const dpp::slashcommand_t& event) {
 		if (event.command.get_command_name() == "set_role") {
 			auto role = std::get<dpp::snowflake>(event.get_parameter("role"));
```

Comment:
Since this is initializing I'd guess this would be `{}` too

Reply:
no

Reply:
I meant the `auto role = `
You're creating a variable `role`

Reply:
Oh lol, yeah I guess it could be